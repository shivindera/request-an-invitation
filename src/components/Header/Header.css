.broccoli-header {
  background: white;
  height: 64px;
  width: 100%;
  padding: 16px;
  border-bottom: 1px solid gray;
  position: fixed;
  top: 0;
  overflow: hidden;
}

.broccoli-header-container {
  display: flex;
  align-items: center;
  width: 100%;
  max-width: 1024px;
  margin: 0 auto;
}

.broccoli-header-logo {
  max-height: 32px;
  width: 100%;
}

.broccoli-header-text {
  font-size: 1.5rem;
  margin-left: 16px;
}
