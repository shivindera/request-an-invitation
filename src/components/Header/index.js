// Static Header component

import logo from '../../images/logo.png';
import './Header.css';

const Header = () => {
  return (
    <header className='broccoli-header'>
      <div className='broccoli-header-container'>
        <a className='broccoli-link' href='/' rel='noopener noreferrer'>
          <img src={logo} className='broccoli-header-logo' alt='logo' />
        </a>
        <h1 className='broccoli-header-text'>BROCCOLI & CO.</h1>
      </div>
    </header>
  );
};

export default Header;
