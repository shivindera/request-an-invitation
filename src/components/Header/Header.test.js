import { render } from '@testing-library/react';
import Header from './';

it('renders Header as expected', () => {
  const header = render(<Header />);
  expect(header.container).toMatchSnapshot();
});
