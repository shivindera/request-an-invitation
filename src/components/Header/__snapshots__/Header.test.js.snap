// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header as expected 1`] = `
<div>
  <header
    class="broccoli-header"
  >
    <div
      class="broccoli-header-container"
    >
      <a
        class="broccoli-link"
        href="/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="logo"
          class="broccoli-header-logo"
          src="logo.png"
        />
      </a>
      <h1
        class="broccoli-header-text"
      >
        BROCCOLI & CO.
      </h1>
    </div>
  </header>
</div>
`;
